warehouse/attestations/services.py

```diff
@@ -151,7 +151,7 @@ def parse_attestations(
         artifact. Attestations are only allowed when uploading via a Trusted
         Publisher, because a Trusted Publisher provides the identity that will be
         used to verify the attestations.
-        Only GitHub Actions Trusted Publishers are supported.
+        Only GitHub Actions and GitLab CI/CD Trusted Publishers are currently supported.
```

Comment:
FWIW, I think we can just remove this line -- this is documented elsewhere and is encoded in `attestation_identity` being present or now anyways.

Reply:
fixed!